modules/kits-elements-defaults/assets/js/commands/create.js

```diff
@@ -0,0 +1,45 @@
+import localSettingsExtractor from '../settings-handlers/local/extract';
+
+export default class Create extends $e.modules.CommandBase {
+	settingsExtractors = [
+		localSettingsExtractor,
+	];
+
+	validateArgs( args = {} ) {
+		this.requireContainer();
+	}
+
+	async apply( { container } ) {
+		$e.internal( 'panel/state-loading' );
+
+		const type = container.model.get( 'widgetType' ) || container.model.get( 'elType' );
+
+		const settings = this.settingsExtractors.reduce( ( carry, extractor ) => {
+			return {
+				...carry,
+				...extractor( container ),
+			};
+		}, {} );
+
+		try {
+			const createAndRefetch = async () => {
+				await $e.data.create( 'kits-elements-defaults/index', { settings }, { type } );
+
```

Comment:
Try `{cache: false}`
```suggestion
			const response = await $e.data.create( 'kits-elements-defaults/index', { settings }, { type } );

```

Reply:
Not working